digitaltwins/device/test/_digital_twin_client_test.js

```diff
@@ -5,9 +5,11 @@
 const assert = require('chai').assert;
 const EventEmitter = require('events');
 const sinon = require('sinon');
+const sandbox = require('sinon').createSandbox();
```

Comment:
Why go for this?  Don't recall it anywhere else.

Reply:
I wanted to use sandbox to make creating and resetting sinon stubs easier. This way I don't worry about restoring sinon stubs after we are done with them, they are sandboxed for each test.

---
In reply to: [403089018](https://github.com/Azure/azure-iot-sdk-node/pull/738#discussion_r403089018) [](ancestors = 403089018)